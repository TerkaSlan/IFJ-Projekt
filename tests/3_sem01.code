/*This test should throw error because of 2 definitions of the same function in same class*/

class Main
{
	static void run()
	{
		ifj16.print("Print called from first Main.run");
	} // end of static void run()
	
} // end of class Main


class Someclass
{
	static void someFunction()
	{
		ifj16.print("Print called from first definition of Someclass.someFunction\n");
	}

	static void someFunction()
	{
		ifj16.print("Print called from second definition of Someclass.someFunction\n");
	}
}