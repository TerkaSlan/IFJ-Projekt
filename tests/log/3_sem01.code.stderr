ERROR: Semantic - Line: 20 - Redefining symbol.
